asf_core_data/pipeline/preprocessing/data_cleaning.py

```diff
@@ -452,7 +454,7 @@ def remove_empty_features(df):
     return df
 
 
-def custom_clean_features(df):
+def custom_clean_features(df, cap_features=False):
```

Comment:
missing description for `cap_features` in docstring.

Reply:
Done, and better explained.